code/modules/events/aurora_caelus.dm

```diff
@@ -28,28 +28,25 @@
 			M.playsound_local(M, 'sound/ambience/aurora_caelus.ogg', 20, FALSE, pressure_affected = FALSE)
 
 /datum/round_event/aurora_caelus/start()
-	for(var/area in GLOB.sortedAreas)
-		var/area/A = area
+	for(var/turf/open/space/S in world)
```

Comment:
considering how many things this iterates over (with a not-quite-cheap proc call), do you think this (along with tick() and end()) might benefit from a CHECK_TICK?

Reply:
The correct answer is yes it should be tick checked. But I also don't want to open Pageant again unless I have to 😎